Update OpenVDB to version 5.2

jay3d opened the pull request.

ras0219-msft committed:
Updates 2019.01.30

UnaNancyOwen committed:
[realsense2] Update to v2.18.0

Update realsense2 port to librealsense v2.18.0.

UnaNancyOwen committed:
[realsense2] Add OpenNI2 driver option

Add OpenNI2 driver option.

jay3d committed:
Update OpenVDB to version 5.2

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=5324) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=5324) before we can accept your contribution.<br/><br/>:x: jay3d [sign now](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=5324)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/vcpkg?pullRequest=5324) it.</sub>

Voskrese commented:
Update to ver. 6.0
https://github.com/Microsoft/vcpkg/pull/5265

jay3d committed:
Merge branch 'master' of https://github.com/Microsoft/vcpkg

renshi-code committed:
Update to freetype-2.9.1 (#5292)

wravery committed:
[cppgraphqlgen] Update to v2.0.0 (#5308)

* [cppgraphqlgen] Update to v2.0.0

* [cppgraphqlgen] Pick up bug fix v2.0.1

* [cppgraphqlgen] Pick up bug fix v2.0.2

ras0219-msft committed:
[rapidjson] PREFER_NINJA

mcraveiro committed:
[vcpkg] Fix warning on Apple (#5333)

Return type of _NSGetExecutablePath is int rather than bool [1],
causing a warning on Apple.

[1] https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/dyld.3.html

zhihaoy committed:
[ampl-mp] Add new port (#5331)

* [ampl-mp] Add new port

This port aims for solver development, so it does not build any
existing solver modules.

The shared library build is disabled for now because it has some
problem on Windows.

closes: #4491

* [ampl-mp] Disable MATLAB MEX support

Squareys committed:
[corrade,magnum,-integration,-extras,-plugins] Update Magnum to v2019.01 (#5312)

* [corrade] Update to v2019.01

Signed-off-by: Squareys <squareys@googlemail.com>

* [magnum] Update to v2019.01, remove shapes and glutapplication features

These features were deprecated in earlier versions and now removed.

Signed-off-by: Squareys <squareys@googlemail.com>

* [magnum-(extras,plugins,integration)] Update to v2019.01, add imgui feat.

Signed-off-by: Squareys <squareys@googlemail.com>

* [magnum] Improve dependencies to corrade

Signed-off-by: Squareys <squareys@googlemail.com>

* [magnum-plugins] Add faad2audioimporter

 - faad2 is currently not in vcpkg, hence commented out
 - The commented out empty line is required to have vcpkg parse the file
   without issues

Signed-off-by: Squareys <squareys@googlemail.com>

jherico committed:
Add spirv-cross to tools directory (#4532)

calumr committed:
[osg] Add Collada model support to OpenSceneGraph (#3950)

* [collada-dom] Initial collada-dom port

* [osg] Add Collada file support

pgoodman committed:
[rocksdb] Add tbb support (#5142)

* Add tbb feature support to rocksdb to enable CLOCK algorithm

* Use RocksDB master branch.

* Undid RocksDB head requirement.

* fix: remove unnecessary FindTbb.cmake

renshi-code committed:
[freetype] fix DLL install on Windows (#5354)

* [wip] fix DLL install on Windows

* change patchfile encoding to UTF-8

sylveon committed:
[detours] new port (#5347)

* [detours] new port

* [detours] Use PATCHES parameter instead of vcpkg_apply_patches function

coryan committed:
Upgrade google-cloud-cpp to 0.6.0. (#5367)

In this version we can disable the unit and integration tests, though it
still tries to find `googletest` so we cannot remove the dependency just
yet.

ras0219-msft committed:
[grpc] Update to v1.18.0
